language/internal/cc/parser/directive.go

```diff
@@ -71,7 +72,11 @@ func (d IncludeDirective) String() string {
 	return fmt.Sprintf("#include \"%s\"", d.Path)
 }
 func (d DefineDirective) String() string {
-	return fmt.Sprintf("#define %s %s", d.Name, strings.Join(d.Tokens, " "))
+	argsString := ""
+	if len(d.Args) >= 0 {
+		argsString = strings.Join(d.Args, ", ")
```

Comment:
`argsString` should probably have the parenthesis here (so `"(" + strings.Join(d.Args, ", ") + ")"`). The format string below can then be `#define %s%s %s`. If there are no arguments, then no parenthesis needed.